packages/python/plotly/plotly/graph_objs/layout/_hoverlabel.py

```diff
@@ -236,11 +236,11 @@ def grouptitlefont(self):
         Sets the font for group titles in hover (unified modes).
         Defaults to `hoverlabel.font`.
 
-        The 'grouptitlefont' property is an instance of Grouptitlefont
+        The 'grouptitlefont' property is an instance of Grouptitle_font
         that may be specified as:
-          - An instance of :class:`plotly.graph_objs.layout.hoverlabel.Grouptitlefont`
+          - An instance of :class:`plotly.graph_objs.layout.hoverlabel.Grouptitle_font`
           - A dict of string/value properties that will be passed
-            to the Grouptitlefont constructor
+            to the Grouptitle_font constructor
 
```

Comment:
@archmoj did this need renaming? I think it's the cause of 
https://github.com/plotly/plotly.py/issues/4820

Reply:
It looks like `grouptitlefont` is still in the Plotly.js schema

Reply:
Oh I thought it's a separate attribute. Thanks for fixing that in #4821.